external/fv3kube/fv3kube/base_yamls/SHiELD/PIRE-like/fv3config.yml

```diff
@@ -0,0 +1,295 @@
+  fv_core_nml:
+    a_imp: 1.0
+    adjust_dry_mass: false
+    beta: 0.0
+    consv_am: false
+    consv_te: 0.0
+    d2_bg: 0.0
+    d2_bg_k1: 0.2
+    d2_bg_k2: 0.1
+    d4_bg: 0.15
+    d_con: 1.0
+    d_ext: 0.0
+    dddmp: 0.5
+    delt_max: 0.002
+    dnats: 1
+    do_vort_damp: true
+    dwind_2d: false
+    ecmwf_ic: false
+    external_ic: false
+    fill: true
+    fill_dp: true
+    fv_debug: false
+    fv_sg_adj: 600
+    gfs_phil: false
+    grid_type: 0
+    hord_dp: -5
+    hord_mt: 5
+    hord_tm: 5
+    hord_tr: -5
+    hord_vt: 5
+    hydrostatic: false
+    io_layout:
+    - 1
+    - 1
+    k_split: 1
+    ke_bg: 0.0
+    kord_mt: 9
+    kord_tm: -9
+    kord_tr: 9
+    kord_wz: -9
+    layout:
+    - 2
+    - 2
+    make_nh: false
+    mountain: true
+    n_split: 6
+    na_init: 0
+    ncep_ic: false
+    nggps_ic: false
+    nord: 3
+    npx: 49
+    npy: 49
+    npz: 79
+    npz_type: gcrm
+    ntiles: 6
+    nudge_qv: true
+    nwat: 6
+    p_fac: 0.05
+    phys_hydrostatic: false
+    print_freq: 1
+    range_warn: true
+    reset_eta: false
+    rf_cutoff: 3000.0
+    rf_fast: true
+    sg_cutoff: 20000.0
+    tau: 5.0
+    tau_h2o: 0.0
+    use_hydro_pressure: false
+    vtdm4: 0.06
+    warm_start: true
+    z_tracer: true
+  gfdl_mp_nml:
+    c_paut: 0.5
+    c_pgacs: 0.01
+    c_psaci: 0.01
+    ccn_l: 300.0
+    ccn_o: 100.0
+    const_vg: false
+    const_vi: false
+    const_vr: false
+    const_vs: false
+    do_cond_timescale: true
+    do_qa: true
+    do_sedi_heat: true
+    do_sedi_w: true
+    dw_land: 0.15
+    dw_ocean: 0.1
+    fix_negative: true
+    icloud_f: 0
+    irain_f: 0
+    liq_ice_combine: true  # Restore default from PIRE simulation codebase (current default is false)
+    prog_ccn: false
+    qi0_crt: 8.0e-05
+    qi_lim: 1.0
+    ql_gen: 0.001
+    ql_mlt: 0.002
+    qs0_crt: 0.003
+    qs_mlt: 1.0e-6
+    rad_graupel: true
+    rad_rain: true
+    rad_snow: true
+    reiflag: 4  # Moved from cld_eff_rad_nml
+    reimax: 150.0  # Moved from cld_eff_rad_nml
+    reimin: 10.0  # Moved from cld_eff_rad_nml
+    rewmax: 10.0  # Moved from cld_eff_rad_nml
+    rewmin: 5.0  # Moved from cld_eff_rad_nml
+    rh_inc: 0.2
+    rh_inr: 0.3
+    rh_ins: 0.3
+    rthresh: 8.0e-6
+    snow_grauple_combine: false  # Restore default from PIRE simulation codebase (current default is true)
+    tau_i2s: 1000.0
+    tau_imlt: 600.0
+    tau_l2v: 300.0
+    tau_v2l: 90.0
+    te_err: 1.0e-14  # Restore default from PIRE simulation codebase (current default is 1.0e-5)
+    vg_max: 16.0
+    vi_fac: 0.85
+    vi_max: 1.0
+    vr_max: 16.0
+    vs_max: 2.0
+    z_slope_ice: true
+    z_slope_liq: true
+  gfs_physics_nml:
+    cal_pre: false
+    cap_k0_land: false
+    cdmbgwd:
+    - 3.5
+    - 0.25
+    cloud_gfdl: true
+    cnvcld: false
+    cnvgwd: true
+    debug: false
+    do_deep: true
+    do_ocean: false
+    dspheat: true
+    fhcyc: 24.0
+    fhlwr: 1800.0
+    fhswr: 1800.0
```

Comment:
Doubling the radiation timestep produces a speedup of about 1.8 simulated years per day when we run on 864 cores (15.5 SYPD vs. 13.7 SYPD).

Reply:
That's right—this is on Gaea.  For context even though I am running SHiELD via the fortran executable on Gaea, I'm still using `fv3kube` and `fv3config` to compile configurations and write run directories for reference runs (I find the "base config" concept useful to help highlight the key namelist parameter changes for each run).  

For recent FV3GFS reference runs, I've used the default radiation timestep in the v0.7 base config, which is 3600 seconds: https://github.com/ai2cm/fv3net/blob/cd7e488c32b3b4e95c0346f0e1a6507eb483f956/external/fv3kube/fv3kube/base_yamls/FV3GFS/v0.7/fv3config.yml#L251-L252

Since it is most true to the original PIRE config, I'll stick with 1800 seconds in this base config, but we can discuss offline what radiation timestep I should use in new coarse reference runs.

Reply:
Thanks, that's helpful and sounds good.